src/ui/features/onboarding/Success/Success.tsx

```diff
@@ -107,8 +111,20 @@ export function Success() {
     },
   });
 
+  const {
+    data: referralProgramEnabled,
+    isLoading: isLoadingRemoteConfigValue,
+  } = useRemoteConfigValue('extension_referral_program');
+
+  if (isLoadingRemoteConfigValue) {
```

Comment:
If we use isLoading, we can set `suspense: false` inside useQuery

Reply:
Thanks, updated ✅